src/Page/Mvc.php

```diff
@@ -10,8 +10,8 @@
 namespace Zend\Navigation\Page;
 
 use Zend\Mvc\ModuleRouteListener;
-use Zend\Mvc\Router\RouteMatch;
-use Zend\Mvc\Router\RouteStackInterface;
+use Zend\Router\Http\RouteMatch;
+use Zend\Router\Http\TreeRouteStack;
```

Comment:
Can you explain, why you use a concrete implementation and no longer the interface?